teeio-validator/library/cxl_ide_test_lib/test_case/ide_km_common.c

```diff
@@ -47,33 +49,39 @@ static bool cxl_ide_generate_key(const void *pci_doe_context,
                                   void *spdm_context, const uint32_t *session_id,
                                   uint8_t stream_id, uint8_t key_sub_stream, uint8_t port_index,
                                   cxl_ide_km_aes_256_gcm_key_buffer_t *key_buffer,
-                                  bool key_iv_gen_capable
+                                  bool key_iv_gen_capable, bool rxdx
```

Comment:
why not use `uint8_t direction`, and use `CXL_IDE_STREAM_DIRECTION_RX` and `CXL_IDE_STREAM_DIRECTION_TX` as input?

That is more aligned to PCIE API - https://github.com/intel/tee-io-validator/blob/main/teeio-validator/library/pcie_ide_test_lib/test_case/ide_km_common.c#L62.

Reply:
Updated.